node/src/components/consensus.rs

```diff
@@ -105,6 +107,7 @@ pub enum Event<I> {
         sender: I,
         proto_block: ProtoBlock,
         timestamp: Timestamp,
```

Comment:
isn't the timestamp now part of the `ProtoBlock`?

Reply:
Thanks! Removed.